src/client/Microsoft.Identity.Client/json/Bson/BsonReader.cs

```diff
@@ -496,8 +496,7 @@ private void ReadType(BsonType type)
                     break;
                 }
                 case BsonType.Binary:
-                    BsonBinaryType binaryType;
```

Comment:
As stated before, we'd like to keep the files in the "json" directory unmodified.